nntrainer/include/databuffer.h

```diff
@@ -42,6 +43,16 @@ namespace nntrainer {
  */
 typedef enum { BUF_TRAIN, BUF_VAL, BUF_TEST, BUFF_UNKNOWN } BufferType;
 
+/**
+ * @brief     Enumeration of data type
+ *            0. DATA_TRAIN ( Data for training )
+ *            1. DATA_VAL ( Data for validation )
+ *            2. DATA_TEST ( Data for test )
+ *            3. DATA_LABEL ( Data for test )
+ *            3. Unknown
```

Comment:
4. ?

Reply:
yes. But because there are PRs which related this, I will change it later. Thanks.